src/__screenshot_tests__/header-screenshot-test.tsx

```diff
@@ -76,3 +76,18 @@ test('Header with truncation and color overrides', async () => {
     const image = await story.screenshot();
     expect(image).toMatchImageSnapshot();
 });
+
+test.each(DEVICES)('Header with bleed', async (device) => {
+    await openStoryPage({
+        id: 'components-headers-header--default',
+        device,
+    });
+
+    const bleedCheckbox = await screen.findByLabelText('Bleed');
+    await bleedCheckbox.click();
```

Comment:
I'd also click on the "Extra content placed on the right in desktop" checkbox to be able to screenshot the `bleed` case in desktop

Reply:
Of course, I wasn't thinking about that. Fixed!